### PR TITLE
feat(api): created api to request eosrate stats by bps

### LINKED ISSUE #90: Include eosrate.io score at the right corner of each bp item.

### PR DESCRIPTION
**Pull request by AngeloCG97**
### What does this PR do?

- Resolve #90 
- [Created api to request eosrate stats by bps](https://github.com/edenia/eden-smart-proxy/commit/3d56eb966623a154174e7d0201813569a131e2f0)
- [Added logic to show eosrate stat](https://github.com/edenia/eden-smart-proxy/commit/a82ef6fc4166c9dda9310ffb002317b2049f5fb3)

### CODE REVIEW

**Comment by AngeloCG97** [comment]
Please do not merge until changes are approved in eosrate and the api is published

**Review by xavier506 — APPROVED** [review]
LGTM

**Pull request merged**
